packages/morph/src/morph.js

```diff
@@ -263,8 +263,8 @@ async function patchChildren(from, to) {
         // Patch elements
         await patch(currentFrom, currentTo)
 
-        currentTo = currentTo && dom(currentTo).next()
-        currentFrom = currentFrom && dom(currentFrom).next()
+        currentTo = currentTo && dom(currentTo).nodes().next()
+        currentFrom = currentFrom && dom(currentFrom).nodes().next()
```

Comment:
Relevant change. When getting the next element for the next loop iteration, we want to use nodes() so we don’t skip text nodes.